dashboard/app/controllers/report_abuse_controller.rb

```diff
@@ -31,57 +31,38 @@ def protected_project?
     project_owner.permission?(UserPermission::PROJECT_VALIDATOR)
   end
 
-  def report_abuse
+  def report_abuse_pop_up
     unless protected_project?
-
       unless verify_recaptcha || !require_captcha?
         flash[:alert] = I18n.t('project.abuse.report_abuse_form.validation.captcha')
-        redirect_to report_abuse_path
-        return
+        return head :forbidden
       end
 
-      unless Rails.env.development? || Rails.env.test?
-        subject = FeaturedProject.featured_channel_id?(params[:channel_id]) ?
-          'Featured Project: Abuse Reported' :
-          'Abuse Reported'
-        response = HTTParty.post(
-          'https://codeorg.zendesk.com/api/v2/tickets.json',
-          headers: {"Content-Type" => "application/json", "Accept" => "application/json"},
-          body: {
-            ticket: {
-              requester: {
-                name: (params[:name] == '' ? params[:email] : params[:name]),
-                email: params[:email]
-              },
-              subject: subject,
-              comment: {
-                body: [
-                  "URL: #{params[:abuse_url]}",
-                  "abuse type: #{params[:abuse_type]}",
-                  "user detail:",
-                  params[:abuse_detail]
-                ].join("\n")
-              },
-              custom_fields: [{id: AGE_CUSTOM_FIELD_ID, value: params[:age]}],
-              tags: (params[:abuse_type] == 'infringement' ? ['report_abuse', 'infringement'] : ['report_abuse'])
-            }
-          }.to_json,
-          basic_auth: {username: 'dev@code.org/token', password: Dashboard::Application.config.zendesk_dev_token}
-        )
-        raise ZendeskError.new(response.code, response.body) unless response.success?
-      end
+      name = current_user&.name || "User not signed in"
+      email = current_user&.email || "User not signed in"
+      age = current_user&.age || "User not signed in"
+      abuse_url = CDO.studio_url(params[:abuse_url]) # reformats url
 
-      if params[:channel_id].present?
-        channel_id = params[:channel_id]
+      send_abuse_report(name, email, age, abuse_url)
+      update_abuse_score
 
-        abuse_score = update_channel_abuse_score(channel_id)
+      return head :ok
+    end
+  end
 
-        update_file_abuse_score('assets', channel_id, abuse_score)
-        update_file_abuse_score('files', channel_id, abuse_score)
+  def report_abuse
+    unless protected_project?
+      unless verify_recaptcha || !require_captcha?
+        flash[:alert] = I18n.t('project.abuse.report_abuse_form.validation.captcha')
+        redirect_to report_abuse_path
+        return
       end
-    end
 
-    redirect_to "https://support.code.org"
+      send_abuse_report(params[:name], params[:email], params[:age], params[:abuse_url])
+      update_abuse_score
+
+      redirect_to "https://support.code.org"
```

Comment:
Just a note that we paired and saw this line needs to be moved down so if the project is protected, then will be redirected to url - hopefully this will fix currently failing UI test.